regtests/ww3_tp2.14/input/prep_env.sh

```diff
@@ -36,7 +37,8 @@ then
      [ "$cmplr" == "datarmor_gnu" ] || [ "$cmplr" == "datarmor_gnu_debug" ]       || \
      [ "$cmplr" == "pgi" ] || [ "$cmplr" == "pgi_debug" ]                         || \
      [ "$cmplr" == "zeus_pgi" ] || [ "$cmplr" == "zeus_pgi_debug" ]               || \
-     [ "$cmplr" == "datarmor_pgi" ] || [ "$cmplr" == "datarmor_pgi_debug" ] ; then
+     [ "$cmplr" == "datarmor_pgi" ] || [ "$cmplr" == "datarmor_pgi_debug" ]       || \
+     [ "$cmplr" == "ukmo_cray_gnu" ] || [ "$cmplr" == "ukmo_cray_gnu_debug" ] ; then
```

Comment:
@ukmo-juan-castillo Could you add an entry for `ukmo_cray` compiler here to keep it consistent with the `cmplr.env` file? I.e:
```
[ "$cmplr" == "ukmo_cray" ] || [ "$cmplr" == "ukmo_cray_debug" ]   || \\
```

Reply:
The issue with the calendar was in matrix.base:

  if [ "$calendar" = 'y' ] && [ "$shrd" = 'y' ]
  then

should be:

  if [ "$calendar" = 'y' ]
  then

because the second condition is redundant. I will update it now that I have to commit again, and also update the prep_env.sh script.